src/Installer/Elastic.Installer.UI/Elasticsearch/Steps/XPackView.xaml.cs

```diff
@@ -92,6 +92,33 @@ protected override void InitializeBindings()
 					this.SkipPasswordGenerationCheckBox.IsEnabled = securityEnabled && canGenerateUsers;
 					
 				});
+
+			this.WhenAnyValue(view => view.ViewModel.ElasticUserPassword)
```

Comment:
You can use `this.WhenAnyValue(x,y,z)` to subscribe to all these at once and have a single `Subscribe`